src/theming.md

```diff
@@ -64,6 +67,8 @@ Key | Type | Description
 `radii` | Array or Object | Values for `borderRadius` props
 `opacity` | Array or Object | Values for `opacity` props
 
+By default arrays of numbers are interpreted as `em`, other unit of measurements can be specified by using an array of strings instead, e.x. `['768px', '992px', '1200px']`.
```

Comment:
This changed in a previous version of styled-system – all numbers are converted to `px` units if they're not strings, but I generally recommend using strings, especially for breakpoints. See https://github.com/jxnblk/styled-system/blob/master/src/index.js#L61

Reply:
I made some updates after i saw your comments. Please review again.

I'm curious - what made you decide that numbers should be converted to px (and not another unit)?

Reply:
Awesome, thanks! The pixel thing come from how React interprets numbers in `style` objects